### PR TITLE
Feature: RSVP to event

### LINKED ISSUE #83: Member: RSVP to event
```
@event
Feature:
  As a member
  In order to tell if I'm attending an event
  I need to RSVP

  Background:
    Given there is a "Local meetup" organization
    And "March 2019 gathering" is scheduled for "2019-03-07"
    And "Jo Johnson" is a member of "Local meetup" organization

  Scenario: Member can RSVP Yes to an event
    Given I'am logged in as "Jo Johnson"
    When I RSVP Yes to "March 2019 gathering"
    Then I will be on a list of interested members for "March 2019 gathering" event

  Scenario: Member can RSVP No to an event
    Given I'am logged in as "Jo Johnson"
    When I RSVP No to "March 2019 gathering"
    Then I will be on a list of members not coming to "March 2019 gathering" event

  Scenario: Member can change RSVP from No to Yes
    Given I'am logged in as "Jo Johnson"
    And I have RSVPed No to "March 2019 gathering" event
    When I change my "March 2019 gathering" RSVP to Yes
    Then I will be on a list of interested members for "March 2019 gathering" event
    And I will not be on a list of members not coming to "March 2019 gathering" event

  Scenario: Member can change RSVP from Yes to No
    Given I'am logged in as "Jo Johnson"
    And I have RSVPed Yes to "March 2019 gathering" event
    When I change my "March 2019 gathering" RSVP to No
    Then I will be on a list of members not coming to "March 2019 gathering" event
    And I will not be on a list of interested members for "March 2019 gathering" event


```

### PR DESCRIPTION
**Pull request by msvrtan**
This PR is adding bit of support on attending/not coming to an event.

I would suggest reviewing commit-by-commit, but please skip first 2 as they are part of #85.

Closes #83

NoteToMyself: rebase after #85 is merged

### CODE REVIEW

File: src/Event/Behat/EventDomainContext.php [hunk 1]
```diff
@@ -44,6 +46,19 @@ public function thereIsOrganization($orgName)
         $this->organization = new OrganizationEntity($orgName, 'Org desc', $this->user, $hometown);
     }
 
+    /**
+     * @Given there is a :orgName organization
+     */
+    public function thereIsOrganization(string $orgName)
```
Comment (zgperak) [inline src/Event/Behat/EventDomainContext.php]: duplicated method

File: src/Organization/Entity/OrganizationEntity.php [hunk 2]
```diff
@@ -81,10 +88,24 @@ public function getFounder(): UserEntity
         return $this->founder;
     }
 
+    public function addOrganizer(UserEntity $user): void
+    {
+        if (in_array($user, $this->organizers)) {
+            throw new \DomainException('This user is already an organizer of this organization');
+        }
+        if (in_array($user, $this->members)) {
```
Comment (HcsOmot) [inline src/Organization/Entity/OrganizationEntity.php]: Same as previous PR - do we need this check?

File: src/Event/Behat/EventDomainContext.php (hunk 1 above)
Comment (msvrtan) [reply src/Event/Behat/EventDomainContext.php]: tnx! fixed

File: src/Organization/Entity/OrganizationEntity.php (hunk 2 above)
Comment (msvrtan) [reply src/Organization/Entity/OrganizationEntity.php]: Lets keep it in the other PR please

**Pull request merged**